launch_testing_ros/examples/check_msgs_launch_test.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2021 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from threading import Event, Thread
+import unittest
+
+import launch
+import launch.actions
+import launch_ros.actions
+import launch_testing.actions
+import pytest
+import rclpy
+from rclpy.node import Node
+from std_msgs.msg import String
+
+
+@pytest.mark.launch_test
+def generate_test_description():
+    return launch.LaunchDescription([
+        launch_ros.actions.Node(
+            package='demo_nodes_cpp',
+            executable='talker',
+            name='demo_node_1',
+        ),
+
+        launch_testing.actions.ReadyToTest()
+    ])
```

Comment:
It would also be good to have an example when we return local variables from the launch testing generation (those are automatically passed to the test cases).
We use that pattern a lot of times, e.g. if you create many processes and need to interact with some of them

Reply:
Updated in the TODO list at https://github.com/ros2/launch/issues/531, will open a new PR with new set of examples !